plans/_integrations_mixed_builders/generic/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG image
+FROM ${image} AS builder
```

Comment:
I am guessing that this is how I would be passing feature flags from the composition file to the `cargo build` command? Works for me. Great.

Reply:
Yes! And we'd use the `PLAN_PATH` below to dispatch between go and rust code.